addons/binding/org.openhab.binding.zway/src/main/java/org/openhab/binding/zway/handler/ZWayDeviceHandler.java

```diff
@@ -466,14 +467,12 @@ private void zwayUnsubscribeOpenHabItem(Item openHABItem) {
 
     @Override
     public void handleUpdate(ChannelUID channelUID, State newState) {
-        // Refresh update time
-        logger.debug("Handle update for channel: {} with new state: {}", channelUID.getId(), newState.toString());
-
-        refreshLastUpdate();
+        // Deprecated, therefore deactivated        
     }
 
     @Override
     public void handleCommand(ChannelUID channelUID, final Command command) {
+        long startTime = System.currentTimeMillis();
```

Comment:
I guess you used the time taken here and debug log to show that your change really improved the speed. Is it still needed now? Otherwise I suggest to remove it again.

Reply:
Yes this was the purpose.
From my side it is okay to remove it but maybe it can also be helpful to identify the reason for a possible latency during command handling.
Or do you think it is not precise enough for that purpose?

Reply:
It might be precise enough but imho it should not be in "production" code. I leave it up to you but my suggestion is to remove it.

Reply:
Yes you are right. I removed it.